Code completion in completionvalues1.xml for <item name="android:alwaysDrawnWithCache">  f^alse  </item> selecting false:
@@ -13 +13
-         <item name="android:alwaysDrawnWithCache">  f^alse  </item>
+         <item name="android:alwaysDrawnWithCache">  false^</item>
